src/main/java/org/openlmis/requisition/domain/RequisitionStatus.java

```diff
@@ -19,6 +19,7 @@
 
 public enum RequisitionStatus {
   INITIATED(1),
+  REJECTED(1),
```

Comment:
What's the consequence of having the same value here? In approval and submitted have different statues - why do these two have the same?